flit_core/flit_core/common.py

```diff
@@ -259,14 +259,14 @@ def check_version(version):
     return version
 
 
-script_template = """\
+script_template = r"""\
```

Comment:
Thanks, but the regular string is necessary for the `"""\<newline>` to work correctly. I think this can stay as is.

Reply:
Ah indeed, upstream doesn't use `"""\`. I thought you wanted to keep more or less in sync with upstream because of #359:
> Update generated scripts to match pip installed scripts

But it's about matching the generated scripts, not necessarily the code that generates them.

Reply:
Yup, that sounds about right. I like having the script contents starting at the start of a line, and this seems like a neat way to do it.